src/plugins/data_table/data_table.js

```diff
@@ -58,6 +58,7 @@ var LocalDataTable = (function() {
       this._enableRowHighlight();
       this.paginate && this._initPaginationHandling();
       this._triggerChangeSelection();
+      this._colReorder && this._colReorder._fnSetupStickyTableHeader.call(this._colReorder, this._colReorder.s.iStickyTableHeader);
```

Comment:
Yay, now that we're working on `tableLoaded` event, we can move it over into the `ColReorder` constructor if the config option is enabled.